internal/core/domain/trade_service.go

```diff
@@ -172,12 +172,7 @@ func (t *Trade) Fail(swapID string, tradeStatus Status, errCode pkgswap.ErrCode,
 }
 
 // AddBlocktime sets the timestamp for a completed trade to the given blocktime.
-// If the trade is not in Complete status, an error is thrown
 func (t *Trade) AddBlocktime(blocktime uint64) error {
-	if !t.IsCompleted() {
```

Comment:
weird, why you had to remove this even if in Settle(), before calling AddBlocktime, you're setting the status to `CompletedStatus`?

Reply:
[This](https://github.com/sekulicd/tdex-daemon/blob/0c75d87e25bb16f1e71dd4c361f28d0f67df2547/internal/core/domain/trade_service_test.go#L31) test was failing, should I revert and update test to use trade.Settle?

Reply:
Updated.